### ISSUE TITLE
FNX3-17092 ⁃ InflateException: SessionControlAdapter Binary XML file line #13: Binary XML file line #13: Error inflating class TextView

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#3539]
https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/5920875

This is happening exclusively on Android 6 - Suspicious.

```
org.xmlpull.v1.XmlPullParserException: Binary XML file line #12: invalid color state list tag gradient
    at android.content.res.ColorStateList.createFromXmlInner(ColorStateList.java:217)
    at android.content.res.ColorStateList.createFromXml(ColorStateList.java:201)
    at android.content.res.Resources.loadColorStateListForCookie(Resources.java:2854)
    at android.content.res.Resources.loadColorStateList(Resources.java:2807)
    at android.content.res.TypedArray.getColor(TypedArray.java:439)
    at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1605)
    at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1584)
    at android.graphics.drawable.VectorDrawable.inflateInternal(VectorDrawable.java:666)
    at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:571)
    at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1215)
    at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1124)
    at android.content.res.Resources.loadDrawableForCookie(Resources.java:2733)
    at android.content.res.Resources.loadDrawable(Resources.java:2643)
    at android.content.res.TypedArray.getDrawable(TypedArray.java:870)
    at android.widget.TextView.<init>(TextView.java:929)
    at android.widget.TextView.<init>(TextView.java:703)
    at androidx.appcompat.widget.AppCompatTextView.<init>(AppCompatTextView.java:3)
    at androidx.appcompat.app.AppCompatViewInflater.createTextView(AppCompatViewInflater.java:2)
    at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:39)
    at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:22)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:746)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
    at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
    at r8.GeneratedOutlineSupport.outline3(outline)
    at org.mozilla.fenix.home.sessioncontrol.SessionControlAdapter.onCreateViewHolder(SessionControlAdapter.kt:1)


    at androidx.recyclerview.widget.RecyclerView$Adapter.createViewHolder(RecyclerView.java:3)
    at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:124)
    at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:9)
    at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1)
    at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:12)
    at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:125)
    at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:8)
    at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:9)
    at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:3)
    at android.view.View.layout(View.java:16694)
    at android.view.ViewGroup.layout(ViewGroup.java:5481)
    at androidx.constraintlayout.motion.widget.MotionController.interpolate(MotionController.java:49)
    at androidx.constraintlayout.motion.widget.MotionLayout.onMeasure(MotionLayout.java:49)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasureChild(CoordinatorLayout.java:1)
    at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasure(CoordinatorLayout.java:33)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
    at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
    at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
    at androidx.appcompat.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:21)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
    at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
    at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
    at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
    at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5995)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
    at com.android.internal.policy.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2662)
    at android.view.View.measure(View.java:18860)
    at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2158)
    at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1262)
    at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1499)
    at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1140)
    at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6233)
    at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
    at android.view.Choreographer.doCallbacks(Choreographer.java:670)
    at android.view.Choreographer.doFrame(Choreographer.java:606)
    at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
    at android.os.Handler.handleCallback(Handler.java:739)
    at android.os.Handler.dispatchMessage(Handler.java:95)
    at android.os.Looper.loop(Looper.java:148)
    at android.app.ActivityThread.main(ActivityThread.java:5527)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:730)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:620)
```

**Comment by NotWoods** [comment]
This is caused by a gradient vector drawable (only supported on Android 7+) being present in the page.

**Comment by abodea** [comment]
Verified as fixed on the latest RC build **`1.0.0-rc.2(Build #11700147 68.0.20190611143747) 6/19.`** with **Lenovo TB3-850M (Android 6).**
As this issue refers to **``SessionControlAdapter``** which refers to home screen where we have the tabs / collections and all the elements but I couldn't reproduce the issue.
